VERSION

```diff
@@ -1 +1 @@
-0.1.1
\ No newline at end of file
+0.1.1-mearns

```

Comment:
It's in setup.py as well, so if you accept the change request, you should remove it in both places.